### DISCUSSION TITLE
Welcome to Discord++ Discussions!

### DISCUSSION THREAD

**Discussion by UndarkAido** [DiscordPP/discordpp#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  The Discord server will always be the best place for quick questions but Discussions should make it easier to track discourse over time.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Wolf6161** [comment]
